coderd/apidoc/docs.go

```diff
@@ -1066,6 +1184,23 @@ const docTemplate = `{
                 }
             }
         },
+        "codersdk.UpdateCheckResponse": {
+            "type": "object",
+            "properties": {
+                "current": {
+                    "description": "Current is a boolean indicating whether the\nserver version is the same as the latest.",
```

Comment:
```suggestion
                    "description": "Boolean indicating whether the\nserver version is the same as the latest.",
```

Reply:
Go comments should/must start with the property name, so I rephrased it into: `Current indicates whether the server version is the same as the latest`.